### PR TITLE
Adding industry classifications rw to coco

### PR DESCRIPTION
**Pull request by nwrigh**

### CODE REVIEW

**Comment by Zapadlo** [comment]
+1

**Comment by catalin-ilea** [comment]
Can you please use the new pattern for sidekicks? https://github.com/Financial-Times/up-service-files/blob/master/public-annotations-api-sidekick%40.service

File: industry-classifications-rw-neo4j-blue-sidekick@.service [hunk 1]
```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Industry Classification Reader-Writer for blue Neo4j Service Sidekick
+BindsTo=industry-classifications-rw-neo4j-blue@%i.service
+After=industry-classifications-rw-neo4j-blue@%i.service
+
+[Service]
+RemainAfterExit=yes
+ExecStart=/bin/sh -c "\
+  export SERVICE=$(echo %p | sed "s/-sidekick//g"); \
```
Comment (catalin-ilea) [inline industry-classifications-rw-neo4j-blue-sidekick@.service]: Unescaped double quotes... @Zapadlo just noticed this on my PR

Comment (catalin-ilea) [inline industry-classifications-rw-neo4j-blue-sidekick@.service]: https://github.com/Financial-Times/up-service-files/pull/510

**Comment by catalin-ilea** [comment]
lgtm

**Pull request merged**